Amazon.QLDB.Driver.IntegrationTests/Amazon.QLDB.Driver.IntegrationTests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net48</TargetFrameworks>
```

Comment:
The object mapper & serialization libraries with target framework: NET 5.0 makes it incompatible with netcoreapp2.1, netcoreapp3.1, net48. 

Need to discuss if we want to force users to update their frameworks to use the object mapper.

Reply:
We can downgrade if we need to. Let's assume we don't need to.